### DISCUSSION TITLE
Feedback on new search behavior: Github webapp co-opting ctrl-f

### DISCUSSION THREAD

**Discussion by incompl** [community/community#70692] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I really like the search features in Github so because this one particular change has been bothering me a lot I wanted to leave feedback.

This is a behavior that I have seen in Github and a couple webapps lately (so it seems it's a bit of a trend) where a webapp will co-opt command-f / ctrl-f to use their in-app search. The unmodified behavior of ctrl-f is reliable because it is implemented in the browser and is the same on every site. Once it is implemented differently on a specific site, you have to learn site-specific behavior just to find and highlight text on a page, something I do probably 100 times a day. For example, the Github one is not dismissed by pressing escape and does not select the current search string when pressing ctrl-f again. My preference wouldn't be for it to be modified to work the same way, because at that point, it may as well not be there. I think the Github search features are good, I just don't want them to override the browser search features so that I can easily use both.

Thanks for reading and for your consideration.

**Comment by Exagone313** [comment]
Would it be possible to add a toggle to disable this feature? I don't want to press CTRL+F twice to use my browser's search features.

The option could be added in this menu on top right:

![23-11-27-15-27-22-177973477](https://github.com/community/community/assets/11819703/42e07c62-9fea-4d04-860a-62fde114265f)

**Reply by parejkoj** [reply]
I went looking for a way to override the browser search hijacking that github does, and could not find it (there is a settings option to configure the command console shortcut, but not the search short). Please at least make the shortcut configurable.

**Comment by curiosity-a** [comment]
You can forbid sites to capture hotkeys. I did a while ago because too many sites misuse it just like you describe(not limited to Ctrl+F either). I only learned from this discussion that there is a replacement search.

But here is the fun part: even if you find a way to get rid of this "feature", builtin search on page doesn't really work on github, because it loads lines of code dynamically. You have to scroll over the entire page first so it fully loads.
